packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/tests/index.test.js

```diff
@@ -7,20 +7,26 @@ import { QueryClient, QueryClientProvider } from 'react-query';
 import userEvent from '@testing-library/user-event';
 import { ThemeProvider, lightTheme } from '@strapi/design-system';
 import { TrackingProvider } from '@strapi/helper-plugin';
+import useAuditLogsData from '../hooks/useAuditLogsData';
 import ListView from '../index';
 import { TEST_PAGE_DATA, TEST_SINGLE_DATA, getBigTestPageData } from './utils/data';
 
 const history = createMemoryHistory();
 const user = userEvent.setup();
 
-jest.mock('@strapi/helper-plugin', () => ({
-  ...jest.requireActual('@strapi/helper-plugin'),
-  useNotification: jest.fn(),
-  useFocusWhenNavigate: jest.fn(),
-  useRBAC: jest.fn(() => ({
-    allowedActions: { canRead: true },
-  })),
-}));
+jest.mock('../hooks/useAuditLogsData', () => jest.fn());
+
+// Codemirror inner dependency, reference: https://github.com/jsdom/jsdom/issues/3002
+// Otherwise it throws: TypeError: range(...).getBoundingClientRect is not a function
+document.createRange = () => {
+  const range = new Range();
```

Comment:
Should we maybe mock these globally in jest instead? Looks like we should either polyfill or mock this everywhere if jsdom doesn't implement the Range API: https://developer.mozilla.org/en-US/docs/Web/API/Range